core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/InstalledIntelliJ.scala

```diff
@@ -73,7 +73,8 @@ sealed abstract class InstalledIntelliJ(root: Path, probePaths: IdeProbePaths, c
 
   private val executable: Path = {
     val content = {
-      val macOsLauncher = paths.root.resolve("MacOS").resolve("idea")
+      val macOsLauncher =
+        paths.root.resolve("MacOS").resolve("idea") // TODO - move "idea" to driver.launch.command ... ?
```

Comment:
don't add todo comments especially if you are unsure. Open an issue and describe the problem.